app/assets/javascripts/discourse/app/models/topic-tracking-state.js

```diff
@@ -410,19 +409,12 @@ const TopicTrackingState = EmberObject.extend({
 
   incrementMessageCount() {
     this.incrementProperty("messageCount");
-    Object.values(this.messageIncrementCallbacks).forEach((cb) => cb());
   },
 
   _generateCallbackId() {
     return Math.random().toString(12).substr(2, 9);
   },
 
-  onMessageIncrement(cb) {
```

Comment:
This was never properly exposed as a plugin API so plugins are not meant to use this function directly so we're removing this without depreciation.